docs/content/integrations/airbyte.mdx

```diff
@@ -0,0 +1,136 @@
+
+### Loading Airbyte assets from YAML config
+
+To load Airbyte assets into Dagster from a set of YAML configuration files, specify the root YAML configuration folder, which contains the `sources`, `destinations`, and `connections` subfolders. Here, the YAML files are treated as the source of truth.
+
+```python startafter=start_load_assets_from_airbyte_project endbefore=end_load_assets_from_airbyte_project file=/integrations/airbyte/airbyte.py dedent=4
+from dagster_airbyte import load_assets_from_airbyte_project
+
+airbyte_assets = load_assets_from_airbyte_project(project_dir="path/to/airbyte/project")
+```
+
+The `load_assets_from_airbyte_project` function parses the YAML metadata, generating a set of software-defined assets which reflect each of the data streams synced by your connections. Each connection has an associated [operation](https://docs.dagster.io/concepts/ops-jobs-graphs/ops#ops) which triggers a sync of that connection.
+
+#### Adding a resource
+
+Assets loaded from Airbyte require an `airbyte_resource`, which defines how to connect and interact with your Airbyte instance.
+
+You can configure this resource and add it to your Airbyte assets by doing the following:
+
+```python startafter=start_airbyte_project_config endbefore=end_airbyte_project_config file=/integrations/airbyte/airbyte.py dedent=4
+from dagster_airbyte import airbyte_resource, load_assets_from_airbyte_project
+
+from dagster import with_resources
+
+airbyte_assets = with_resources(
+    load_assets_from_airbyte_project(project_dir="path/to/airbyte/project"),
+    {
+        "airbyte": airbyte_resource.configured(
+            {
+                "host": "localhost",
+                "port": "8000",
+            }
+        )
+    },
+)
+```
+
+### Loading Airbyte assets from Airbyte instance
+
+To load Airbyte assets into Dagster from a live Airbyte instance, you will need to configure an Airbyte resource which defines how to connect to that instance. Here, the Airbyte instance is treated as the source of truth.
+
+```python startafter=start_load_assets_from_airbyte_instance endbefore=end_load_assets_from_airbyte_instance file=/integrations/airbyte/airbyte.py dedent=4
+from dagster_airbyte import airbyte_resource, load_assets_from_airbyte_instance
+
+airbyte_instance = airbyte_resource.configured(
+    {
+        "host": "localhost",
+        "port": "8000",
+    }
+)
+airbyte_assets = load_assets_from_airbyte_instance(airbyte_instance)
+```
+
+The `load_assets_from_airbyte_instance` function retrieves a list of connection definitions via the Airbyte API, generating a set of software-defined assets which reflect each of the data streams synced by your connections. Each connection has an associated [operation](https://docs.dagster.io/concepts/ops-jobs-graphs/ops#ops) which triggers a sync of that connection.
+
+The assets produced by this function are automatically bound the passed `airbyte_resource` to use at runtime.
```

Comment:
I think this will go over a lot of readers' heads. Is there a way of phrasing it that would appeal to someone with a somewhat less technical background?

Reply:
I've taken another pass at the wording, hopefully this is a bit easier to understand

Reply:
Words like "bound" and "runtime" might be unfamiliar. More clear might be "you don't need to use `with_resources` to bind an Airbyte resource to the assets produced by `load_asset_from_airbyte_instance`, because the Airbyte resource supplied to it will already be bound to them".